packages/db/migrations/0093.do.speech.sql

```diff
@@ -0,0 +1,28 @@
+-- Type: DO
+-- Name: speech
+-- Description: Add speech table containing text to speech audio_url and speech_marks
+
+BEGIN;
+
+CREATE TABLE omnivore.speech (
+    id uuid PRIMARY KEY DEFAULT uuid_generate_v1mc(),
+    user_id uuid NOT NULL REFERENCES omnivore.user ON DELETE CASCADE,
+    elastic_page_id TEXT NOT NULL,
+    voice text,
+    audio_url text NOT NULL,
```

Comment:
If we did do the signed URLs we'd have to store this differently, we'd probably have to store the bucket + file path and then create a signed URL when we return.

Reply:
Yeah. I will also add a `state` field